tests/testthat/test-stats-htest.R

```diff
@@ -19,6 +19,8 @@ test_that("tidy.htest/oneway.test", {
   gl <- glance(ot)
   
   check_tidy_output(td, strict = FALSE)
+  expect_true(!("num df" %in% colnames(td)))
```

Comment:
```suggestion
  expect_false("num df" %in% colnames(td))
```

Reply:
Would minorly improve readability since you don't have to mentally parse the negative